### PR TITLE
Allow generating photon noise tables for rav1e

### PR DESCRIPTION
**Pull request by shssoichiro**
Note that the `--photon-noise-table` flag in rav1e is currently behind the `unstable` feature, so rav1e must be compiled with `--features unstable` for this to be supported.

### CODE REVIEW

File: av1an-core/src/scenes.rs [hunk 1]
```diff
@@ -336,6 +336,23 @@ fn validate_zones_args() {
     .contains(&"--aq-mode=1".to_owned()));
 }
 
+#[test]
+fn validate_rav1e_zone_with_photon_noise() {
+  let input = "45 729 rav1e --speed 6 --photon-noise 4";
```
Comment (CartoonFan) [inline av1an-core/src/scenes.rs:341]: I'm not an expert, but the tests seem to suggest "rav1e reset" instead of just "rav1e"?

**Review by CartoonFan — APPROVED** [review]
The descriptive changes look fine. I don't know anything about the actual code, though.

**Review by BlueSwordM — APPROVED** [review]
Everything seems good, although it should be noted that the only way to get grain synth inputs to work with rav1e are to build it with "unstable" features.

**Review by master-of-zen — APPROVED** [review]
Amazing:) :+1: :1st_place_medal:

**Pull request merged**